test/com/amazon/ion/impl/IonReaderContinuableTopLevelBinaryTest.java

```diff
@@ -608,13 +694,12 @@ public void multipleSymbolTablesBetweenValues(boolean constructFromBytes) throws
             constructFromBytes
         );
 
-        nextExpect(IonType.SYMBOL);
-        expectString("abc");
-        nextExpect(IonType.SYMBOL);
-        expectString("def");
-        nextExpect(IonType.SYMBOL);
-        expectString("purple");
-        nextExpect(null);
+        assertSequence(
+            next(IonType.SYMBOL), stringValue("abc"),
+            next(IonType.SYMBOL), stringValue("def"),
+            next(IonType.SYMBOL), stringValue("purple"),
```

Comment:
Why not use `symbolValue(...)`? Same elsewhere.

Reply:
`stringValue` and `symbolValue` test different reader APIs. `IonReader.stringValue` is applicable to both symbols and strings, so we test it for both. `IonReader.symbolValue` is applicable only to symbols.